src/Grafana/dashboards/aspnetcore.json

```diff
@@ -1304,16 +1316,17 @@
         "type": "query"
       },
       {
+        "allValue": ".*",
```

Comment:
This is the `instance` template variable - it as all of the same changes as the `job` template variable.